src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java

```diff
@@ -55,6 +65,16 @@ public void testInvalidConfig() {
                 response.getBody().jsonPath().getString("errors[0]"));
     }
 
+    @Ignore
+    @Test
+    public void test51CharsFileName() {
+        Response response = postNewBundle(filenameWith51CharsJson);
+
+        assertEquals(200, response.getStatusCode());
+        assertEquals(Constants.STITCHED_FILE_NAME_FIELD_LENGTH_ERROR_MSG, response.getBody().jsonPath().getString(
```

Comment:
what was the plan with this? are you going to handle this issue separately, the fact that we're returning 200 for a validation error

Reply:
Will be done as part of https://tools.hmcts.net/jira/browse/EM-3603